hedera-node/hedera-smart-contract-service-impl/src/test/java/com/hedera/node/app/service/contract/impl/test/exec/systemcontracts/hts/customfees/TokenCustomFeesTranslatorTest.java

```diff
@@ -53,9 +57,14 @@ void setUp() {
 
     @Test
     void matchesTokenCustomFeesTranslatorTest() {
-        given(attempt.selector()).willReturn(TokenCustomFeesTranslator.TOKEN_CUSTOM_FEES.selector());
-        final var matches = subject.matches(attempt);
-        assertThat(matches).isTrue();
+        given(attempt.isSelector(any(Function.class))).willReturn(true);
```

Comment:
Rewrite of this unit test (and the next added test) is kind of weak.  You're mocking the only real functionality being tested.  Prefer to actually test the method being tested (in this case, since it's a one-liner deferring to another method - test that method) by providing actually matching and mismatching selectors.

Reply:
Reworked the mocking of the attempt to be specific for each call.